qbft/instance_store.go

```diff
@@ -0,0 +1,84 @@
+package qbft
```

Comment:
this should be in testing utils not here

Reply:
why testing? 
i can separate the interface and the implementation if that what you mean